main.js

```diff
@@ -127,9 +127,9 @@ const menu = Menu.buildFromTemplate(template)
 Menu.setApplicationMenu(menu)
 
 const TOP = 30;
-const TOTAL_WIDTH = 2000;
+const TOTAL_WIDTH = 1500;
 const TOTAL_HEIGHT = 1000;
-const LEFT_WIDTH = 1000;
+const LEFT_WIDTH = 750;
```

Comment:
This is fine, but I'm doing to make it just take 90% of the screen width in my branch.